src/templates/Article/teasers.js

```diff
@@ -537,7 +537,8 @@ const createTeasers = ({
         inline: !node.data.grid,
         ActionBar: ActionBar,
         PayNote: PayNote,
-        Link: Link
+        Link: Link,
+        t: t
```

Comment:
I know this is a black magic shitty problem but we need to do it slightly differently to work in Publikator where props fn is not used. Inject it via the component function without making it part of the `props` function and object:
```
    component: ({ ...props }) => {
      return <SeriesNav t={t} {...props} />
    },
```

Reply:
ok